spark-indexer/src/test/java/org/uniprot/store/spark/indexer/chebi/ChebiRDDReaderTest.java

```diff
@@ -0,0 +1,46 @@
+package org.uniprot.store.spark.indexer.chebi;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ResourceBundle;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.junit.jupiter.api.Test;
+import org.uniprot.core.cv.chebi.ChebiEntry;
+import org.uniprot.store.spark.indexer.common.JobParameter;
+import org.uniprot.store.spark.indexer.common.util.SparkUtils;
+
+import scala.Tuple2;
+
+/**
+ * @author lgonzales
+ * @since 10/05/2020
+ */
+class ChebiRDDReaderTest {
+
+    @Test
+    void testLoadChebi() {
+        ResourceBundle application = SparkUtils.loadApplicationProperty();
+        try (JavaSparkContext sparkContext = SparkUtils.loadSparkContext(application)) {
+            JobParameter parameter =
+                    JobParameter.builder()
+                            .applicationConfig(application)
+                            .releaseName("2020_02")
+                            .sparkContext(sparkContext)
```

Comment:
releaseName `2020_02 ` should be parametrized

Reply:
it is part of the path in resource dir..